### PR TITLE
Problems with Database.java with getting and creating assignments.

### PR DESCRIPTION
**Pull request by mahimashanware**
I've set this up to be very similar to the comments functionality on the trainings, but with a level of abstraction for the exact database functions. I've been trying to debug but I am unsure right now as to what the problem in my code is. The AssignmentServlet is called when on the assignments.html page, and should be automatically showing the current assignments and let the user create new assignments, both of which are failing.

### CODE REVIEW

**Comment by ecao1107** [comment]
Judging from the "entity" page, there is already an assignment entity in DataStore, is it created by the Servlet or you created it manually?
https://pantheon.corp.google.com/datastore/entities;kind=Assignment;ns=__$DEFAULT$__/query/kind?project=step25-2020

**Comment by mahimashanware** [comment]
I created that one manually, but I've been able to create and view assignments now :)

File: project/src/main/java/com/google/autograder/servlets/AssignmentServlet.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+/** Servlet that encapsulates the subtraction game. */
+@WebServlet("/assignment")
+public final class AssignmentServlet extends HttpServlet {
+
+  private Database d = new Database();
```
Comment (BenLin00) [inline project/src/main/java/com/google/autograder/servlets/AssignmentServlet.java:22]: nit: maybe a more descriptive variable name?

**Review by BenLin00 — DISMISSED** [review]
Looks good to me

**Review by BenLin00 — APPROVED** [review]
LGTM

**Pull request merged**